app/script/calling/CallMessageMapper.js

```diff
@@ -30,33 +30,33 @@ z.calling.CallMessageMapper = (function() {
    * @param {Object} event - Call event object
    * @returns {CallMessage} Call message entity
    */
-  const _map_event = function(event) {
-    const {content: call_message, conversation: conversation_id, from: user_id, sender: client_id, time} = event;
+  const mapEvent = function(event) {
+    const {content: callMessage, conversation: conversationId, from: userId, sender: clientId, time} = event;
 
-    const additional_properties = {
-      client_id: client_id,
-      conversation_id: conversation_id,
+    const additionalProperties = {
+      clientId: clientId,
+      conversationId: conversationId,
       time: time,
-      user_id: user_id,
+      userId: userId,
     };
 
     let content = undefined;
-    switch (call_message.type) {
+    switch (callMessage.type) {
       case z.calling.enum.CALL_MESSAGE_TYPE.GROUP_SETUP:
       case z.calling.enum.CALL_MESSAGE_TYPE.UPDATE: {
-        const {dest_clientid, dest_userid, props: properties, sdp} = call_message;
+        const {destClientid, destUserid, props: properties, sdp} = callMessage;
```

Comment:
These values are agreed upon between clients and are part of the protocol specification. Thus they cannot be changed.